cli/tests/data/bundle/package_no_match_schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "packagingVersion": "3.0",
+  "name": "bitbucket",
+  "version": "4.5",
+  "releaseVersion": 3,
```

Comment:
We should remove this property from the `bundle-schema` and the `package-schema`. This is controlled by Universe and Cosmos.